docs/Docker/dockerLocal.md

```diff
@@ -67,7 +68,7 @@ The environment file (`.env`) now contains the MPS and RPS environment variables
 
 
 
-2. To check all containers are up and running run the following command.
+2. Check that all containers are running:
 
     ```bash
     `docker ps --format 'table{{.Image}}\t{{.Status}}\t{{.Names}}'`
```

Comment:
Can you also drop the extra quotes around this command, conflicts with copy/pasting